src/schema-error.ts

```diff
@@ -0,0 +1,142 @@
+// adapted from https://github.com/apideck-libraries/better-ajv-errors (MIT)
```

Comment:
I did not use the code directly as:
- I was getting NPE for some cases
- I am not sure if it makes sense to bring the other libraries that they use for suggestions
- We can improve some error messages later if needed

Reply:
I don't particularly like this, but I'll let it slip :smile: